README.md

```diff
@@ -132,34 +133,35 @@ The new technical design aims to overcome the problems listed above.
+- Implement authentication according to the principles of the [Web Authentication API](https://www.w3.org/TR/webauthn/). Don't implement authentication in the browser TLS session establishment layer using TLS CCA. Special TLS CCA configuration is no longer required in the web servers which simplifies using eID cards in cloud services and clustered systems.
 - Require HTTPS and reject requests from non-secure contexts.
-- No PIN caching and no need to restart the browser after logout.
+- No need to restart the browser after logout as the PIN is prompted during each authentication.
 - Use the same mechanism for authentication and signing.
+- Use the same, consistent visuals for authentication and signing across browsers and operating systems.
 - Support most common European eID smart cards.
 
-The following diagram provides an overview of the new Web eID solution components:
+The following diagram provides an overview of the Web eID solution components (in orange). The Safari Web Extension component model differs slightly, though not in principle, see the corresponding diagram in section *[Safari](#safari)* below.
 
 ![Web eID components](diagrams/Web-eID-Component-Model.png)  
 Figure 3: Web eID components
 
 ## Requirements for the new solution
 
-Besides the requirements listed above, the new solution satisfies the following requirements:
+Besides the principles listed above, the new solution satisfies the following requirements:
 
-1. Support authentication and digital signing with Estonian, Latvian, Lithuanian, Finnish eID smart cards in phase 1 and [**TODO:** who else? Austria, **Belgium**, Germany, Italy, Liechtenstein, Portugal, Spain, Serbia, Croatia, Czech Republic, Slovakia, see overview [here](https://ec.europa.eu/cefdigital/wiki/display/CEFDIGITAL/Country+Overview+-+eID)] and Yubikey in phase 2.
-1. Support recent versions of common operating systems, Windows, Mac OS X, Linux, and browsers, Chrome, Firefox, Safari, Opera, Edge, Internet Explorer.
-1. Support easy installation and immediate launch through the browser without requiring restart of the operating system or browser after installation. Note that the Web eID native application needs to be installed separately from the browser extension.
-1. Does not require any other third party software besides the operating system standard PC/SC stack and standard card reader driver.
+1. Support authentication and digital signing with Estonian, Latvian, Finnish (tier 1) and Croatian, Lithuanian (tier 2) eID smart cards in phase 1 and other EU eID cards in next phases.
+1. Support recent versions of common operating systems, Windows, macOS, Linux, and evergreen browsers, Chrome, Edge, Firefox, Safari.
```

Comment:
https://thingsthemselves.com/reminder-safari-is-not-an-evergreen-browser/

Reply:
Thanks, good point! I changed the sentence slightly: _and evergreen browsers Chrome, Edge, Firefox, and Safari._ Hope it comes through to a knowledgeable reader.